### PR TITLE
Dockerfile for building images for Docker and Kubernetes.

### PR DESCRIPTION
**Pull request by tobiasake**
### Description of the Change
Adding a Dockerfile so user can build Docker image for Docker and Kubernetes.

### Benefits
Make it easy to build RemRem-Generate docker images.

### Sign-off

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.

Signed-off-by:

### CODE REVIEW

File: service/src/main/docker/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,9 @@
+FROM tomcat:8.0-jre8
+LABEL maintainer Eiffel-Community 
+ARG URL
+RUN echo Bulding RemRem-Publish image based on artifact url: ${URL}
```
Comment (e-backmark-ericsson) [inline service/src/main/docker/Dockerfile]: Should be 'Generate' right?

Comment (tobiasake) [reply service/src/main/docker/Dockerfile]: Fixed.

File: wiki/markdown/docker.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+  
+  Windows: https://docs.docker.com/docker-for-windows/install/
+
+
+## Follow these step to build the Docker image.
+
+1. Change to service directory: 
+`cd service`
+
+1. Execute maven package command to build the RemRem-Generate war file:
```
Comment (xvinosi-github) [inline wiki/markdown/docker.md]: Update the number?

File: service/src/main/docker/README.txt [hunk 3]
```diff
@@ -0,0 +1,16 @@
+A: Build RemRem-Generate Docker image based on RemRem-Generate artifact from an Artifactory, e.g. Jitpack:
+cd (git root dir)/service
+docker build -t remrem-generate:0.10.8 --build-arg URL=https://jitpack.io/com/github/Ericsson/eiffel-remrem-generate/generate-service/0.10.8/generate-service-0.10.8.war -f src/main/docker/Dockerfile .
```
Comment (e-pettersson-ericsson) [inline service/src/main/docker/README.txt]: Is the url correct with Ericsson in it? Shouldn't it be eiffel-community?

File: wiki/markdown/docker.md [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+`docker build -t remrem-generate:0.10.9 --build-arg URL=./target/generate-service-0.10.9.war -f src/main/docker/Dockerfile .` 
+
+
+Now docker image has build with tag "remrem-generate:0.10.9"
+
+## Run Docker image on local Docker Host
+To run the produced docker image on the local Docker host, execute this command: 
+
+
+`docker run -p 8080:8080 --expose 8080 -e server.port=8080 -e logging.level.log.level.root=DEBUG -e logging.level.org.springframework.web=DEBUG -e logging.level.com.ericsson.ei=DEBUG remrem-generate:0.10.9`
```
Comment (e-pettersson-ericsson) [inline wiki/markdown/docker.md]: Should default port be set to 8080, when publish also has the default port set to 8080? Maybe use the same ports as in eiffel-sandbox so they can easily be started up simultaneously?

File: wiki/markdown/docker.md (hunk 2 above)
Comment (tobiasake) [reply wiki/markdown/docker.md]: Fixed.

File: wiki/markdown/docker.md (hunk 4 above)
Comment (tobiasake) [reply wiki/markdown/docker.md]: Fixed.

File: service/src/main/docker/README.txt (hunk 3 above)
Comment (tobiasake) [reply service/src/main/docker/README.txt]: Fixed.

**Pull request merged**